types/onfleet__node-onfleet/onfleet.d.ts

```diff
@@ -9,31 +9,43 @@ import Teams = require('./Resources/Teams');
 import Webhooks = require('./Resources/Webhooks');
 import Workers = require('./Resources/Workers');
 
+interface BottleneckOptions {
+    /** default 20 */
+    LIMITER_RESERVOIR?: number;
+    /** default 10000 */
+    LIMITER_WAIT_UPON_DEPLETION?: number;
+    /** default 1 */
+    LIMITER_MAX_CONCURRENT?: number;
+    /** default 50 */
+    LIMITER_MIN_TIME?: number;
+}
+
 declare class Onfleet {
-  apiKey: string;
-  api: {
-    baseUrl: string;
-    timeout: number;
-    headers: {
-      'Content-Type': string;
-      'User-Agent': string;
-      Authorization: string;
-    }
-  };
+    apiKey: string;
+    api: {
+        baseUrl: string;
+        timeout: number;
+        headers: {
+            'Content-Type': string;
+            'User-Agent': string;
+            Authorization: string;
+        };
+    };
 
-  constructor(api_key: string);
-  verifyKey(): Promise<boolean>;
+    constructor(api_key: string, timeout?: number, bottleneckOptions?: BottleneckOptions);
```

Comment:
@marcobeltempo - Can we add also this param `baseURL?: string` to the constructor. I just merged a this [PR](https://github.com/onfleet/node-onfleet/pull/46) to master in order to include this new param in @onfleet/node-onfleet. Thanks!

Reply:
Thanks for the heads up @efgallegos. I've gone ahead and added the optional `baseURL` param [(fa2c511)](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/55000/commits/fa2c5114b5e695199c3b87c06b5616748c4d8905)